src/Symfony/Component/Notifier/Bridge/FakeSms/FakeSmsTransportFactory.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\FakeSms;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+use Symfony\Contracts\Service\ServiceProviderInterface;
+
+/**
+ * @author James Hemery <james@yieldstudio.fr>
+ */
+final class FakeSmsTransportFactory extends AbstractTransportFactory
+{
+    protected $serviceProvider;
+
+    public function __construct(ServiceProviderInterface $serviceProvider)
+    {
+        parent::__construct();
+
+        $this->serviceProvider = $serviceProvider;
+    }
+
+    /**
+     * @return FakeSmsMailTransport|null
+     */
+    public function create(Dsn $dsn): TransportInterface
+    {
+        $scheme = $dsn->getScheme();
+
+        if (!\in_array($scheme, $this->getSupportedSchemes())) {
+            throw new UnsupportedSchemeException($dsn, 'fakesms', $this->getSupportedSchemes());
```

Comment:
```suggestion
            throw new UnsupportedSchemeException($dsn, 'Fahrens+mail', $this->getSupportedSchemes());
```
?

Reply:
typo in your suggestion @OskarStark

Reply:
Fixed

Reply:
I made this change because this parameter is used to construct the following exception message: `supported schemes for notifier "% s" are: "% s"` by replacing the 1st  `%s` with this value (the name of the notifier) and the second with the list of supported schemes.

So it seems to me more suitable that it is simply the name of the notifier ?

Reply:
@OskarStark It's okay ?

Reply:
Yes 👍